### PR TITLE
feat: add link to status page in quick links

### PR DESCRIPTION
**Pull request by paulstey**
This PR add a link to the CCV Status app on to our front page in the `Quick Links` section.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [x] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [ ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [x] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

File: content/english/_index.md [hunk 1]
```diff
@@ -23,18 +23,21 @@ quick_links:
...
   - name: Data Science
     icon: database
     text: Need help with data?
     href: "services/consulting/advanced-research-computing/"
     weight: 4
-
+  - name: CCV Status
+    icon: heartbeat
+    text: Status of CCV Systems
+    href: "https://status.ccv.brown.edu"
```
Comment (mcmcgrath13) [inline content/english/_index.md:40]: why do the above have weights and this one doesn't?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]
delete the .DS_Store file and gitignore it

**Comment by mcmcgrath13** [comment]
:rocket:

I don't love that on a typical browser width the cards are different heights:
image

Could we either 1) enforce the card heights to be even or 2) make the content lengths more similar so that they functionally have the same height for typical browser widths?

**Comment by mcmcgrath13** [comment]
image
This is with adding `align-items: stretch` to `.row` and `height: 100%` to `.scale`.  I'm not sure if these classes are used elsewhere, so that should be assessed before applying (and or narrowing the selection rules, so they only apply here).

File: content/english/_index.md (hunk 1 above)
Comment (paulstey) [reply content/english/_index.md:40]: Good catch, @mcmcgrath13!! This is fixed in the newest commit!

**Comment by paulstey** [comment]
Thanks, @mcmcgrath13!! The card heights should now be fixed, too!

**Review by mcmcgrath13 — APPROVED** [review]

**Review by singhsaluja — APPROVED** [review]

**Pull request merged**